src/components/Header/ListOfContents.js

```diff
@@ -1,21 +1,122 @@
-export default function ListOfContents(props) {
+import "./Header.css";
+import {
+  FaPhone,
+  FaHome,
+  FaListAlt,
+  FaUserCircle,
+  FaWindowRestore,
+} from "react-icons/fa";
+import { motion } from "framer-motion";
+
+export default function ListOfContents() {
+  const listVariants = {
+    visible: { transition: { staggerChildren: 0.1 } },
+  };
+
+  const itemVariants = {
+    hidden: { opacity: 0, scale: 0.5 },
+    visible: {
+      opacity: 1,
+      scale: 1,
+      transition: {
+        type: "spring",
+        damping: 15,
+        stiffness: 200,
+      },
+    },
+  };
   return (
-    <ul>
-      <li onClick={props.hideNav}>
-        <a href="#home">Home</a>
-      </li>
-      <li onClick={props.hideNav}>
-        <a href="#about">About</a>
-      </li>
-      <li onClick={props.hideNav}>
-        <a href="#skills">Skills</a>
-      </li>
-      <li onClick={props.hideNav}>
-        <a href="#projects">Projects</a>
-      </li>
-      <li onClick={props.hideNav}>
-        <a href="#contact">Contact</a>
-      </li>
-    </ul>
+    <div className="nav_menu" id="nav-menu">
+      <motion.ul
+        variants={listVariants}
+        initial="hidden"
+        animate="visible"
+        className="nav_list"
+        id="navlist"
+      >
+        <motion.li className="nav_item" variants={itemVariants}>
+          <motion.div whileHover={{ scale: 1.2 }} whileTap={{ scale: 1 }}>
+            <a href="#" className="nav_link active">
+              <FaHome style={{ marginRight: "20px" }} />
+              Home
+            </a>
+          </motion.div>
+        </motion.li>
+        <motion.li className="nav_item" variants={itemVariants}>
+          <motion.div whileHover={{ scale: 1.2 }} whileTap={{ scale: 1 }}>
+            <a href="#about" className="nav_link">
+              <FaUserCircle style={{ marginRight: "20px" }} />
+              About
+            </a>{" "}
+          </motion.div>
+        </motion.li>
+        <motion.li className="nav_item" variants={itemVariants}>
+          <motion.div whileHover={{ scale: 1.2 }} whileTap={{ scale: 1 }}>
+            <a href="#skills" className="nav_link">
+              <FaListAlt style={{ marginRight: "23px" }} />
+              Skills
+            </a>{" "}
+          </motion.div>
+        </motion.li>
+        <motion.li className="nav_item" variants={itemVariants}>
+          <motion.div whileHover={{ scale: 1.2 }} whileTap={{ scale: 1 }}>
+            <a href="#work" className="nav_link">
+              <FaWindowRestore style={{ marginRight: "5px" }} />
+              Projects
+            </a>{" "}
+          </motion.div>
+        </motion.li>
+        <motion.li className="nav_item" variants={itemVariants}>
+          <motion.div whileHover={{ scale: 1.2 }} whileTap={{ scale: 1 }}>
+            <a href="#contact" className="nav_link">
+              <FaPhone style={{ marginRight: "7px" }} />
```

Comment:
Avoid at any cost using inline style due to the fact that the specificity and CSS Conflicts: Inline styles have high specificity, which means they can override external CSS rules. This can lead to unexpected behavior and conflicts that are difficult to debug. External stylesheets follow a more predictable cascading order and specificity rules. 
Also you need to somehow make it consistent and not to have different values on margin. 
Lessons: ->
https://css-tricks.com/specifics-on-css-specificity/
https://www.youtube.com/watch?v=CHyPGSpIhSs